### ISSUE TITLE
Create docker entrypoint for auto create database on the first build

### ISSUE THREAD

**Issue by Keda87** [Keda87/django-simple-banking-api#3]
The goal is to make this project fully automated when running with docker.

Currently, we should run these following commands to make this project runs.
```
Then create the database and do migrations.

$ docker-compose exec bank_db createdb --username=postgres db_banking
$ docker-compose exec bank_api python manage.py migrate
$ docker-compose exec bank_api python manage.py collectstatic
```

**Comment by adityaarakeri** [comment]
can take this up. assign this to me.

**Comment by Keda87** [comment]
@adityaarakeri sure, thanks!

**Comment by ThirumlaDevi** [comment]
@adityaarakeri Will it be okay if I pick this up, if you are not working on it? I can help @Keda87  with this as i've done similar automations like [this](https://github.com/dreikanter/feeder/pull/263) before.
CC: @Keda87

**Comment by Keda87** [comment]
Hi @ThirumlaDevi yeah, sure.
will assign this to you and thank you very much!

**Comment by ThirumlaDevi** [comment]
@Keda87 I have raised #4 which contains an automised setup to start, stop and run tests over dockerized banking service.

**Comment by ThirumlaDevi** [comment]
@Keda87 Could you please let me know if the above automation is what you are looking for? If not we can work towards it. TIA.

**Comment by Keda87** [comment]
Hi @ThirumlaDevi, sorry I'm not noticed your PR, lets discuss this on the PR.
and very appreciated for your work!

**Comment by Keda87** [comment]
Due to this ticket is already implemented on PR #4 , so I close this issue.
Thank you @ThirumlaDevi, much appreciate your work!
